tasks/main.yml

```diff
@@ -36,19 +41,14 @@
 - name: Copy Varnish configuration (systemd).
   template:
     src: varnish.params.j2
-    dest: /etc/varnish/varnish.params
+    dest: "{{ varnish_config_path }}/varnish.params"
     owner: root
     group: root
     mode: 0644
   when: >
     (ansible_os_family == 'RedHat' and ansible_distribution_major_version|int >= 7) or
     (ansible_os_family == 'Debian' and ansible_distribution_release == "xenial")
 
-- name: Ensure Varnish config path exists.
-  file:
-    path: "{{ varnish_config_path }}"
```

Comment:
I think we should still ensure the config path dir exists, in case it's not /etc/varnish.